__tests__/components/Table.spec.js

```diff
@@ -361,6 +361,81 @@ describe('Table component', () => {
 
       expect(cellsText).toEqual(dataText)
     })
+
+    it('should update the table when data changes', () => {
+      const field = 'Name'
+      const cols = [
+        {
+          field
+        }
+      ]
+
+      const data = [
+        {
+          Name: 'Caterpie',
+          description: 'lorem ipsum dolor sit amet'
+        },
+        {
+          Name: 'Blastoise',
+          description: 'consecteur adipiscing elit, sed do eiusmod'
+        }
+      ]
+
+      const component = mount(
+        <Table columns={cols} data={data} />
+      )
+
+      data.push({
```

Comment:
Would it make sense to also check the rendered component before the push as well?
Might be redundant though since we probably already check that elsewhere.